plugins/docker/docker_

```diff
@@ -319,155 +509,22 @@ def main():
         sys.exit(1)
 
     if mode == "suggest":
-        print("cpu")
-        print("images")
-        print("memory")
-        print("network")
-        print("status")
-        print("volumes")
+        print("multi")
```

Comment:
I like your approach to change the default to *multi* (while keeping the single graphs working). But please add a comment here, why only one of the available wildcard features is announced.
Your future readers (including yourself) will surely appreciate this :)